extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -527,6 +527,7 @@ export class AmpAutocomplete extends AMP.BaseElement {
       return Promise.resolve();
     }
     if (typeof src === 'string') {
+      this.srcBase_ = src;
```

Comment:
Should this be done immediately or once remote data is fetched/promise is resolved below?

Reply:
Immediately, since this is used in the subsequent call: 
https://github.com/ampproject/amphtml/blob/fde6ea6b9c01fd04103adcef5078220a739f1f70/extensions/amp-autocomplete/0.1/amp-autocomplete.js#L405